h/api/schemas.py

```diff
@@ -226,9 +223,11 @@ def validate(self, data):
         new_appstruct['text'] = appstruct.pop('text', u'')
         new_appstruct['tags'] = appstruct.pop('tags', [])
 
+        permissions = appstruct.pop('permissions', None)
+
         # Replace the client's complex permissions object with a simple shared
         # boolean.
-        if appstruct.pop('permissions')['read'] == [new_appstruct['userid']]:
+        if permissions and permissions['read'] == [new_appstruct['userid']]:
```

Comment:
This is an out of date diff but the current behaviour is still to create public annotations if no permissions are given.

We could make it default to private instead?

When hacking around on the command line, since all the fields are optional, it's really easy to create public annotations currently because you don't have to pass any perms. If we changed it to private by default you'd have to pass the pretty annoying permissions object to make a public annotation. 

On the other hand we'll be able to make the API accept a simple shared boolean in the future.

And maybe people hacking around with the API ought to be doing it in private by default!

So I'm happy to change it to private by default.

@nickstenning ?